tracer/test/snapshots/MultipartFormTests.ApiWebRequestMultipartTest.verified.txt

```diff
@@ -0,0 +1,11 @@
+﻿--faa0a896-8bc8-48f3-b46d-016f2b15a884
+Content-Type: application/json
+Content-Disposition: form-data; name="Name 1"; filename="FileName 1.json"
```

Comment:
Hmmm, does `filename` need to be URL encoded? 🤔 I'm guessing so, and judging by the `HttpClient` implementation?